.appveyor.yml

```diff
@@ -80,8 +80,11 @@ install:
   # install tox
   - python -m pip install -r dev-requirements.txt
 
-  # install meson and ninja for autohintexe
-  - call %PYTHON3%\python.exe -m pip install meson ninja
+  # install meson and ninja for autohintexe.
+  # we install meson from master because it includes a 'meson.exe' entry point
+  # that makes it easier to run meson on Windows when installed via pip:
+  # https://github.com/mesonbuild/meson/pull/4004
+  - %PYTHON3%\python.exe -m pip install git+https://github.com/mesonbuild/meson.git#egg=meson ninja
```

Comment:
This should be `- call %PYTHON3%\python.exe …`.

Reply:
i put it back, it's weird though